static/js/main.js

```diff
@@ -218,14 +218,11 @@ function captureFrame() {
 
 function addScreenshot(screenshot) {
     window.shark.screenshots[screenshot.id] = screenshot;
-
     let time_input_value = convertUnixSecsForInput(screenshot.time);
 
     let html = `
         <div class='screenshot media-metadata-container'>
-            <div class='image-area media-area' width='${screenshot.width}' height='${screenshot.height}'>
-                <img src='${screenshot.dataURL}' />
-            </div>
+            <canvas class='image-area media-area' width='${screenshot.width}' height='${screenshot.height}'></canvas>
```

Comment:
Why the substantial change from just an `img` to a `canvas`?
Generally, we don't want to display canvas if we can just do an image.
(There are several reasons, but accessibility is the top one.)

This also messes with the styling.

Reply:
I'm not sure I understand, don't we need to use a canvas if we want to display an image that can optionally be flipped?

Reply:
A canvas does not need to be attached to anything to work.
See:
https://github.com/linqs/shark-clipper/blob/main/static/js/main.js#L340
That canvas is created, used, and tossed when done.